packages/server/README.md

```diff
@@ -68,6 +68,13 @@ npm run test-watch ./test/unit/api_spec.coffee
 
 To run an individual e2e test:
 
+You have to run `packages/runner` in order to be able to execute the e2e tests.
+
+```bash
+cd ..\runner
+npm run watch
+```
+
```

Comment:
Not true, you just need to have the front-end packages (`driver`, `runner`, `reporter`, `extension`, `desktop-gui`...) built before running the e2e tests.

`npm install` should do this for you. `npm run build` from the root will also do this for you, as would `npm run watch`.

I'd probably update this to just say "You must build all packages before running the E2E tests. Run `npm run build` or `npm run watch` to accomplish this.

Reply:
Ah you're right. I was able to execute the tests after building all.
To make it clear that those commands must be executed from the prtoject root, I slightly adjusted your suggestion:

"Run `npm run build` or `npm run watch` in the project root to accomplish this."